doc/whats_new/v0.22.rst

```diff
@@ -217,6 +217,8 @@ Changelog
     values both for training and predicting. They also support infinite
     values. :pr:`13911` and :pr:`14406` by `Nicolas Hug`_, `Adrin Jalali`_
     and `Olivier Grisel`_.
+  - |MajorFeature| Estimators now support :term:`sample_weight`. :pr:`14696` by
+    `Adrin Jalali`_ and `Nicolas Hug`_.
```

Comment:
This is major? No strong opinion.

Maybe we can just make the whole GBDT compound entry as major (and change MajorFeature to just Feature here and for the missing values)